pkgs/development/libraries/v8/default.nix

```diff
@@ -124,13 +124,7 @@ stdenv.mkDerivation rec {
 
   # Patch based off of:
   # https://github.com/cowboyd/libv8/tree/v5.1.281.67.0/patches
-  patches = lib.optional (!doCheck) ./libv8-5.4.232.patch
-  ++ stdenv.lib.optionals stdenv.isDarwin [ ./no-xcode.patch ];
```

Comment:
Guess maybe it's not worth maintaining this, but in general I think we should try to avoid xcbuild (if it's reasonable to do so) and stay as close as possible to the linux builds.

Reply:
Yeah, my thinking here is less patches, less maintenence. In chromium-based things like v8 & node, xcode is used but only for picking up the sdk version, I think. Ideally we wouldn't need it at all but that's kind of upstream's problem.